src/Views/listings/search.phtml

```diff
@@ -86,7 +74,7 @@ $title = "Search page";
             <?php endforeach; ?>
         <?php else:?>
             <div class="text-center listing-search-result">
-                <h3>No results for "<?= $params['query'] ?>"</h3>
+                <h3>No results for "<?= $params['query']?>"<h3>
```

Comment:
The / in the \<h3\> closing tag is missing. Also add the space back in at the end of the \<?= [...] ?\> block